openstack/cce/v1/nodes/list.go

```diff
@@ -0,0 +1,62 @@
+package nodes
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack"
+)
+
+// List returns collection of nodes.
+func List(client *golangsdk.ServiceClient, clusterID string) (*ListNodes, error) {
+	raw, err := client.Get(fmt.Sprintf("https://%s.%s", clusterID, client.ResourceBaseURL()[8:])+
+		strings.Join([]string{"nodes"}, "/"), nil, openstack.StdRequestOpts())
```

Comment:
There is literally no sense in joining single-element slice :smile: 
Also, generating this URL takes two rows, it's better to move it into variable in this case:
```go
listURL := fmt.Sprintf("https://%s.%s/nodes/", clusterID, client.ResourceBaseURL()[8:]))
raw, err := client.Get(listURL, ...
```